### ISSUE TITLE
[pull-kubevirt-e2e-k8s-1.21-sig-network] macvtap e2e test setup is flaky on CI

### ISSUE THREAD

**Issue by EdDev** [kubevirt/kubevirt#6500]
/kind bug

**What happened**:
macvtap e2e test setup is flaky on CI

Occurrences are rare (once a week): https://search.ci.kubevirt.io/?search=Macvtap+VMI+migration+with+live+traffic+should+keep+connectivity+after+a+migration&maxAge=336h&context=1&type=bug%2Bjunit&name=&excludeName=&maxMatches=5&maxBytes=20971520&groupBy=job

Example:
```
 13:24:28: • Failure in Spec Setup (BeforeEach) [236.303 seconds]
13:24:28: [sig-network] [Serial]Macvtap
13:24:28: tests/network/framework.go:27
13:24:28:   VMI migration
13:24:28:   tests/network/vmi_multus.go:1286
13:24:28:     with live traffic
13:24:28:     tests/network/vmi_multus.go:1308
13:24:28:       should keep connectivity after a migration [BeforeEach]
13:24:28:       tests/network/vmi_multus.go:1367
13:24:28: 
13:24:28:       should have managed to figure out the IP of the server VMI
13:24:28:       Unexpected error:
13:24:28:           <*errors.errorString | 0xc00038ad00>: {
13:24:28:               s: "timed out waiting for the condition",
13:24:28:           }
13:24:28:           timed out waiting for the condition
13:24:28:       occurred
13:24:28: 
13:24:28:       tests/network/vmi_multus.go:1360 
```

Ref: https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/6488/pull-kubevirt-e2e-k8s-1.21-sig-network/1443556290562363392#1:build-log.txt%3A2877

**What you expected to happen**:
Not seeing this unstable error.

**How to reproduce it (as minimally and precisely as possible)**:
Unknown

**Anything else we need to know?**:

**Environment**:
Current master, on the pull-kubevirt-e2e-k8s-1.21-sig-network job.

**Comment by EdDev** [comment]
This seems to be a low priority issue as it occurs only once a week and on an experimental CNI (macvtap).
